File: dropDatabase.py
import mysql.connector
from mysql.connector import errorcode

try:
    #Connecting to the database using mysql.connector.
    db = mysql.connector.connect(host="localhost",
                                 user="Kreutz",
                                 database="gmat",
                                 password="#Mustangs18PennState18")

    cur = db.cursor()

    cur.execute("""drop table characterData""")
    cur.execute("""drop table monsterData""")
    cur.execute("""drop table simpleMeleeWeapons""")
    cur.execute("""drop table simpleRangedWeapons""")
    cur.execute("""drop table martialMeleeWeapons""")
    cur.execute("""drop table martialRangedWeapons""")



#Test for errors.
except mysql.connector.Error as err:
    #Check for correct login info.
    if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
        print("Incorrect Login Information.. Please Try Again.")
    #Check for if database exists.
    elif err.errno == errorcode.ER_BAD_DB_ERROR:
        print("Invalid Database. Aborting..")
    #Print an error.
    else:
        print(err)

#Close connection.
else:
    db.close()

File: dropDatabase.php
<?php

    include 'dbconnect.php';

$characterData = "drop table characterData";
$monsterData = "drop table monsterData";
$smw = "drop table simpleMeleeWeapons";
$srw = "drop table simpleRangedWeapons";
$mmw = "drop table martialMeleeWeapons";
$mrw = "drop table martialRangedWeapons";
$startingArea = "drop table startingArea";
$passageWidth = "drop table passageWidth";
$passage = "drop table passage";
$doorType = "drop table doorType";
$chamber = "drop table chamber";
$chamberExits = "drop table chamberExits";
    
$tables = [$characterData, $monsterData, $smw, $srw, $mmw, $mrw, $startingArea, $passageWidth, $passage, $doorType, $chamber, $chamberExits];
$count = 1;    
 foreach($tables as $x => $sql){
    if(mysqli_query($connection, $sql)) {
        echo "Table $count Dropped <br>";
    } else {
        echo "Error Dropping Table $x <br>";
    }
    $count++;
 }
    mysqli_close($connection);
?>